typescript/sdk/src/HyperlaneApp.ts

```diff
@@ -26,11 +26,22 @@ export class HyperlaneApp<
     super(connectedContractsMap);
   }
 
-  static fromAddressesMap<F extends HyperlaneFactories>(
+  getContracts(chain: ChainName): HyperlaneContracts<Factories> {
+    return this.get(chain);
+  }
+
+  getAddresses(chain: ChainName): HyperlaneAddresses<Factories> {
+    return serializeContracts(this.get(chain));
+  }
+
+  static fromAddressesMapHelper<F extends HyperlaneFactories>(
```

Comment:
Why not just keep this named `fromAddressesMap` and have subclasses call `super`? The new name seems a bit odd for for a public method

Reply:
iirc `super.static` is invalid

Reply:
but agree avoiding rename is nicer
I presume this is not a widely used interface but appreciate the sentiment that we should avoid breaking changtes

Reply:
Good point but they could access it via the class' name directly

Reply:
I don't think that works, as super will return a HyperlaneApp instance (as opposed to the subclass).

Know any ways around this?

Reply:
Sorry I wasn't clear. I mean access the `HyperlaneApp` static directly like this: 
```
  static fromAddressesMap(
    addressesMap: HyperlaneAddressesMap<any>,
    multiProvider: MultiProvider,
  ): HyperlaneIsmFactory {
    const fromAddressesMap = HyperlaneApp.fromAddressesMap(
      addressesMap,
      ismFactoryFactories,
      multiProvider,
    );
```

Reply:
No you were clear, I was replying to the original comment as the others hadn't loaded for me

Reply:
Okay tbh I should have just looked at this closer before suggesting a solution but on second look here's what I'm really trying to say:
There's no reason for `fromAddressesMapHelper` to be in HyperlaneApp and I recommend just making it a util function somewhere

Reply:
Works for me